platform/src/main/resources/db/migration/V6__sep24_field_updates.sql

```diff
@@ -0,0 +1,59 @@
+ALTER TABLE sep24_transaction ADD amount_expected VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD kyc_verified VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD message VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD more_info_url VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refund_memo VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refund_memo_type VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD refunded BOOLEAN;
+
+ALTER TABLE sep24_transaction ADD refunds JSON;
+
+ALTER TABLE sep24_transaction ADD request_asset_code VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD request_asset_issuer VARCHAR(255);
+
+ALTER TABLE sep24_transaction ADD required_info_message VARCHAR(255);
```

Comment:
@lijamie98

Reply:
This field should not be in `JdbcSep24Transaction`. 

We should move the field out from the JdbcSepTransaction to JdbcSep31Transaction and regenerate the migration file.

Reply:
.